siff-dr-client/client-netfilter.go

```diff
@@ -13,8 +33,33 @@ func processPackets() {
 	}
 
 	for packet := range nfq.GetPackets() {
+		log.Println("Adding SIFF headers")
+		var empty []byte
+		var empty2 []byte
+		var flags layers.IPv4Flag
+		flags |= IS_SIFF
```

Comment:
Should this be setting the EXP bit too? Packets seems to be getting dropped at the first hop